Model/Menu/Node/Image/File.php

```diff
@@ -123,7 +123,11 @@ public function getImageSize(string $file): array
         $mediaDirectory = $this->filesystem->getDirectoryRead(DirectoryList::MEDIA);
         $fileFullPath = $mediaDirectory->getAbsolutePath(self::PATH . $file);
 
-        return getimagesize($fileFullPath);
+        try {
+            return getimagesize($fileFullPath);
+        } catch (Exception $e) {
+            return [];
```

Comment:
what is the point of returning empty array as code calling this is expecting array with information - throw proper exception to be handled in application logic